### ISSUE TITLE
Reworking the converter

### ISSUE THREAD

**Issue by LzLang** [LzLang/Zendro-Converter#16]
On the BrAPI Hackathon we separated the data model definitions, due the separation the converter needs to be reworked/changed to work with the new models.

In the original models were primary keys already existing but we "ignored" those and used our own primary keys and this has led to redundant information/attributes.
We would like to unify/merge those primary keys to prevent unnecessary/redundant attributes.
-> We plan to use the names of BrAPIs primary keys but you will still be able to use a custom name for those.

In some files are multiple models due the separation, therefore we need to consider this while walking through a file.

Currently is `get_items` a rather large function, it would be beneficial (probably) to separate it into multiple functions to increase readability and clarity.

Also we would like to implement arrays and use those in our data definitions.

So basically we want to ensure the function and efficiency of the converter by revising it.
Also we are planing to write a small BrAPI data converter to convert existing data (from an other project) to fit our models, so we can upload existing data to create a warehouse (feeding Cassavebase datasets into Zendro).
Beneath working on a converter we thought about working with generic data models in the future, but for this we need to work on Zendro (beneficial for linking directly to existing databases but on the contrary you can't add your own data, therefore it's currently just an idea).

**Comment by LzLang** [comment]
Version 1 of the converter was finalized in the past.
The new version works on different BrAPI-Models, which are officially onedimensionalised.

New models: https://github.com/plantbreeding/BrAPI/tree/data-model-separation/Specification/BrAPI-Schema

New converter: https://github.com/LzLang/Zendro-Converter/tree/new_converter
